tests/post/features/example_solution.feature

```diff
@@ -0,0 +1,27 @@
+@local @solution
+Feature: Example solution
```

Comment:
Feature is not "Example solution", but "Solutions CLI" or just "Solutions", file should be also renamed accordingly.